spec/javascripts/mobile/mobile_comments_spec.js

```diff
@@ -82,14 +82,8 @@ describe("Diaspora.Mobile.Comments", function(){
   describe("createComment", function () {
     beforeEach(function() {
       spec.loadFixture("aspects_index_mobile_post_with_comments");
-      var commentBoxHtml = spec.fixtureHtml("comments_mobile_commentbox");
       var link = $(".stream .comment-action").first();
       Diaspora.Mobile.Comments.showCommentBox(link);
-      jasmine.Ajax.requests.mostRecent().respondWith({
```

Comment:
True, it's a shim for the most recent comments ajax request. I'm surprised we don't break any tests by removing the shim. @svbergerem?